builder.json

```diff
@@ -78,6 +78,27 @@
                         "-DBUILD_TESTING=OFF"
                     ]
                 }
+            },
+            "hosts": {
+                "alpine": {
+                    "!packages": [
+                        "maven",
+                        "openjdk8"
+                    ],
+                    "!build_steps": [
+                        "mvn -P continuous-integration -B compile",
+                        "build-classifier"
```

Comment:
oh builder.json ... you confusing scamp

are we using `targets/hosts/alpine` to override the overrides from `target/linux/armvX`, to put stuff back to its default values so it actually runs tests ??? 🫠

if the old linux-armv6/7/64 CI jobs ACTUALLY ran their tests, all these overrides wouldn't be necessary.

but we don't have aws-crt-builder docker images for armv6/7, so it's not a simple fix

just ranting...

Reply:
yeah, I have added a comment.